### PR TITLE
config.py: Limit render test in test room

### PR DESCRIPTION
**Pull request by mohitkyadav**
Add ACCESS_CONTROLS for render test command

Closes https://github.com/coala/corobo/issues/359

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -49,4 +49,6 @@
 
 CHATROOM_PRESENCE = os.environ.get('ROOMS', '').split() or ROOMS_TO_JOIN
 
+ACCESS_CONTROLS = {'render test': {'allowrooms': ('coala/cobot-test',)}, }
+
```
Comment (Vamshi99) [inline config.py]: Add `coala/corobo` room also

**Comment by jayvdb** [comment]
Can you check with @meetmangukiya about this.  He build the `utils/filters.py` which is doing something a bit similar.  Would be interesting to know why he needed to build a filter to do that.

**Comment by meetmangukiya** [comment]
> Can you check with @meetmangukiya about this. He build the utils/filters.py which is doing something a bit similar. Would be interesting to know why he needed to build a filter to do that.

Seems like I wrote something that already existed 😓

**Comment by meetmangukiya** [comment]
ack c7b3784e2732a247dd11b2e6e63f1ce888e01b95

**Comment by meetmangukiya** [comment]
@gitmate-bot rebase

**Comment by meetmangukiya** [comment]
@gitmate-bot rebase

**Comment by meetmangukiya** [comment]
@mohitkyadav rebase this and we can merge it :)

**Comment by meetmangukiya** [comment]
@gitmate-bot rebase

**Comment by sils** [comment]
@gitmate-bot rebase please for debugging

**Comment by sils** [comment]
@gitmate-bot rebase for debug

**Comment by mohitkyadav** [comment]
@sils rebased.

**Comment by jayvdb** [comment]
@rultor merge

**Pull request merged**